internal/store/builder.go

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
Any way we can preserve the history of this one? it's really just a rename right?

Reply:
It's not just a rename, it also renames the individual build functions to Store suffix, e.g. `buildDaemonSetCollection` -> `buildDaemonSetStore `

guthub is bad at displaying the diff, but I split it multiple commits, so hopefully, it's easier. :)

Reply:
Ah I see. In that case the diff is probably too large for git to acknowledge, that this might be a rename, so it would have had to happen in two commits. In that case I'm ok with it as is.